README.md

```diff
@@ -305,6 +306,88 @@ RESET_PASSWORD_SECRET_KEY: The secret key for reset password functionality.
+
+### Overview
+
+Deployment is streamlined through **Vercel**, with dedicated buttons for the **Frontend** and **Backend** applications. Both require specific configurations during and after deployment to ensure proper functionality.
+
+---
+
+### Frontend Deployment
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvintasoftware%2Fnextjs-fastapi-template%2Ftree%2Fmain%2Fnextjs-frontend&env=API_BASE_URL&envDescription=The%20API_BASE_URL%20is%20the%20backend%20URL%20where%20the%20frontend%20sends%20requests.)
+
+1. **Deploying the Frontend**  
+   - Click the **Frontend** button above to start the deployment process.  
+   - During deployment, you will be prompted to set the `API_BASE_URL`. Use a placeholder value (e.g., `http://`) for now, as this will be updated with the backend URL later.  
+   - Complete the deployment process.
+
+2. **Post-Deployment Configuration**  
+   - Navigate to the **Settings** page of the deployed frontend project.  
+   - Access the **Environment Variables** section.  
+   - Update the `API_BASE_URL` variable with the backend URL once the backend deployment is complete.
+
+---
+
+### Backend Deployment
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvintasoftware%2Fnextjs-fastapi-template%2Ftree%2Fmain%2Ffastapi_backend&env=CORS_ORIGINS,TEST_DATABASE_URL,DATABASE_URL,ACCESS_SECRET_KEY,RESET_PASSWORD_SECRET_KEY,VERIFICATION_SECRET_KEY)
+
+1. **Deploying the Backend**  
+   - Click the **Backend** button above to begin deployment.  
+   - During the deployment process, you will be prompted to configure the following environment variables:
+
+     - **CORS_ORIGINS**  
+       - Set this to `["*"]` initially to allow all origins. You will update this with the frontend URL later.
+
+     - **DATABASE_URL**  
+       - Use a placeholder value (e.g., `https://`) if you don't yet have the actual database URL. Replace it with the correct URL post-deployment.
+
+     - **ACCESS_SECRET_KEY**, **RESET_PASSWORD_SECRET_KEY**, **VERIFICATION_SECRET_KEY**  
+       - You can temporarily set these secret keys as plain strings (e.g., `examplekey`) during deployment. However, you should generate secure keys and update them after the deployment in the **Post-Deployment Configuration** section.
+
+   - Complete the deployment process.
+
+2. **Post-Deployment Configuration**  
+   - Access the **Settings** page of the deployed backend project.  
+   - Navigate to the **Environment Variables** section and update the following variables with secure values:
+
+     - **CORS_ORIGINS**  
+       - Once the frontend is deployed, replace `["*"]` with the actual frontend URL.
+
+     - **DATABASE_URL**  
+       - Update this with the actual database URL if not set during deployment.
+
+     - **ACCESS_SECRET_KEY**  
+       - Generate a secure key for API access and set it here.  
```

Comment:
Did you add somewhere in the readme the script I sent you to generate secret keys? It's worth mentioning it in this secion.

Reply:
yes, there's one section showing that
https://github.com/vintasoftware/nextjs-fastapi-template/pull/58/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R121